integration/gotest_test.go

```diff
@@ -55,6 +55,7 @@ func testPkgs(t *testing.T) []string {
 		"github.com/u-root/u-root/cmds/core/tail",
 		"github.com/u-root/u-root/cmds/core/wget",
 		"github.com/u-root/u-root/cmds/core/which",
+		"github.com/u-root/u-root/cmds/core/uptime",
```

Comment:
If there's no testdata file, this doesn't need to be in the blacklist

Reply:
Thats true.I removed the testdata files so I will remove the uptime command  from that list.